### PR TITLE
dev-db/influxdb: version bump

### PR DESCRIPTION
**Pull request by nemunaire**
Hi!

Here is a version bump for InfluxDB; ebuild's modifications relate to update of dependencies.

### CODE REVIEW

**Comment by nemunaire** [comment]
cc @williamh

File: dev-db/influxdb/influxdb-1.7.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	"golang.org/x/sys d8e400bc7db4870d786864138af681469693d18c github.com/golang/sys"
+	"golang.org/x/text f21a4dfb5e38f5895301dc265a8def02365cc3d0 github.com/golang/text"
+	"golang.org/x/time 26559e0f760e39c24d730d3224364aef164ee23f github.com/golang/time"
+	"google.golang.org/genproto fedd2861243fd1a8152376292b921b394c7bef7e github.com/google/go-genproto"
+	"google.golang.org/grpc 168a6198bcb0ef175f7dacec0b8691fc141dc9b8 github.com/grpc/grpc-go"
+	)
+
+inherit golang-build golang-vcs-snapshot systemd user
+
+DESCRIPTION=" Scalable datastore for metrics, events, and real-time analytics"
```
Comment (Dessa) [inline dev-db/influxdb/influxdb-1.7.6.ebuild]: whitespace

**Comment by juippis** [comment]
The maintainer doesn't seem to be using Github. You'd have better chance of reaching them via opening a bug in Gentoo's bugzilla.

**Comment by Dessa** [comment]
superseded by #12586

**Comment by juippis** [comment]
Indeed and there seems to be multiple bugs open about bumping this as well. Please continue talk in he PR linked above, and/or https://bugs.gentoo.org/689880

Sorry that the maintainer didn't notice this earlier.

**Pull request closed without merging**